lib/xml_builder.ex

```diff
@@ -231,16 +232,19 @@ defmodule XmlBuilder do
   defp first_element(element_spec),
     do: element(element_spec)
 
-  defp format_content(children, level) when is_list(children),
-    do: ['\n', Enum.map_join(children, "\n", &format(&1, level))]
+  defp format_content(children, level, options) when is_list(children),
+    do: ['\n', Enum.map_join(children, "\n", &format(&1, level, options))]
 
-  defp format_content(content, _level),
+  defp format_content(content, _level, options),
     do: escape(content)
 
   defp format_attributes(attrs),
     do: Enum.map_join(attrs, " ", fn {name,value} -> [to_string(name), '=', quote_attribute_value(value)] end)
 
-  defp indent(level),
+  defp indent(_, [indent: false]),
```

Comment:
ok, I'll modify the original function to look for the option in that way, and remove this one.

Reply:
I'm adding the other indentation options discussed since I'm here anyway. But, let me run this by you:  instead of having format: :indent and then also having an indentation option to set the character, why not just have format: :none as an option, and then indentation: "\t" as the other?  I'll push this code so you can see what I mean.  doing that eliminates one condition and maintains the result.